src/marginaldamage3.jl

```diff
@@ -1,68 +0,0 @@
-include("fund.jl")
```

Comment:
Can we for now not rename these files? Just makes it more difficult to review the changes here.